components/core/Settings/Jumper.js

```diff
@@ -0,0 +1,412 @@
+  ${Styles.BUTTON_RESET};
+  color: ${theme.semantic.textGray};
+`;
+
+const STYLES_JUMPER_MAIN = css`
+  ${Styles.HORIZONTAL_CONTAINER};
+  justify-content: space-between;
+  align-items: flex-start;
+  width: 100%;
+  height: 348px;
+  order: 1;
+`;
+
+const STYLES_JUMPER_SIDEBAR = css`
+  ${Styles.HORIZONTAL_CONTAINER};
+  align-items: flex-start;
+  width: 190px;
+  height: 100%;
+  padding: 20px 0px;
+  border-right: 1px solid ${Constants.system.grayLight5};
+`;
+
+const STYLES_JUMPER_CONTENT = css`
+  ${STYLES_FLEX_COLUMN}
+  align-items: flex-start;
+  width: 100%;
+  height: 340px;
+  overflow: scroll;
+  margin: 0px 20px;
+  padding-bottom: 160px;
+  padding-top: 20px;
+  ::-webkit-scrollbar {
+    width: 0;
+    background: transparent; 
+  }
+`;
+
+const STYLES_ACCOUNT = css`
+  ${STYLES_FLEX_COLUMN}
+  align-items: center;
+  width: 100%;
+`;
+
+const STYLES_ACCOUNT_USER = css`
+  ${STYLES_FLEX_COLUMN}
+  align-items: center;
+  margin: 10px 0px;
+`;
+
+const STYLES_ITEM_CONTAINER = css`
+  ${STYLES_FLEX_COLUMN}
+  width: 100%;
+  height: 76px;
+  margin-top: 24px;
+`;
+
+const STYLES_ITEM_HEADER = css`
+  font-size: 12px;
+  margin-bottom: 8px;
+  color: ${Constants.system.gray};
+`;
+
+const STYLES_AVATAR_BOX = css`
+  ${STYLES_FLEX_COLUMN}
+  width: 100%;
+  height: 76px;
+`;
+
+const STYLES_AVATAR_UPLOAD = css`
+  ${Styles.HORIZONTAL_CONTAINER};
+  align-items: flex-end;
+`;
+
+const STYLES_EDIT_INFO_FOOTER = css`
+  display: flex;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  width: 448px;
+  height: 50px;
+  padding: 12px 20px;
+  background-color: ${Constants.system.white};
+  border-radius: 0px 0px 16px 0px;
+  border-top: 1px solid ${Constants.system.grayLight5};
+`;
+
+const STYLES_MENU_CONTAINER = css`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  width: 200px;
+  height: auto;
+`;
+
+const STYLES_MENU_ITEM = css`
+  display: flex;
+  font-weight: 400;
+  font-size: 14px;
+  width: 100%;
+  height: 32px;
+  border-radius: 12px;
+  padding: 8px;
+  :hover {
+    background-color: ${Constants.system.grayLight5};
+    cursor: pointer;
+  }
+`;
+
+const STYLES_MENU_ITEM_ICON = css`
+  display:flex;
+  width: 16px;
+  height: 16px;
+  margin-right: 16px;
+`;
+
+const STYLES_FILE_HIDDEN = css`
+  height: 1px;
+  width: 1px;
+  opacity: 0;
+  visibility: hidden;
+  position: fixed;
+  top: -1px;
+  left: -1px;
+`;
+
+function truncateString(str, num) {
+  if (str.length > num) {
+    return str.slice(0, num) + "...";
+  } else {
+    return str;
+  }
+}
+
+const Sidebar = (props) => {
+  return (
+    <div css={STYLES_JUMPER_SIDEBAR}>
+      <div css={STYLES_ACCOUNT}>
+        <ProfilePhoto
+          user={props.viewer}
+          style={{ marginBottom: "4px", borderRadius: '10px' }}
+          size={48}
+        />
+        <div css={STYLES_ACCOUNT_USER}>
+          <p style={{ fontWeight: "500", fontSize: "14px" }}>{props.viewer.name}</p>
```

Comment:
this should be `<SYSTEM.H5 style={{ marginBottom: "2px"}}>{props.viewer.name}</SYSTEM.H5>`